src/NoTargets/Sdk/Sdk.props

```diff
@@ -10,9 +10,12 @@
     <UsingMicrosoftNoTargetsSdk>true</UsingMicrosoftNoTargetsSdk>
   </PropertyGroup>
 
+  <!-- For CPS/VS support. Importing in .props allows any subsequent targets to redefine this if needed -->
+  <Target Name="CompileDesignTime" />
```

Comment:
Please move this to be after the `CustomBeforeNoTargetsProps` import

Reply:
That would mean the target would not be overridable by that targets, say. Is that intentional?

Reply:
Fixed anyway, it's not like targets are normally defined in props anyway...